apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -494,6 +496,26 @@
     });
   };
 
+  LockScreenStateManager.prototype.resumeEvents =
+  function lssm_resumeEvents(resumeEvents) {
+    resumeEvents.forEach((ename) => {
+      if (this.suspendedEvents[ename]) {
+        this.suspendedEvents[ename] = false;
+        window.addEventListener(ename, this);
+      }
+    });
+  };
+
+  LockScreenStateManager.prototype.suspendEvents =
```

Comment:
You can use the method comments to describe the similar thing that I've commented at line 85. And plus why 'holdcamera' is the only event we now deal with (because hardware event will be fired even after the window get hidden).